Shevchuk_Maksym/README.md

```diff
@@ -0,0 +1,93 @@
+Discord Registration and Usage Guide
+Introduction
+Discord is a popular communication platform that enables users to chat, voice call, and video call with individuals and groups. This document provides a step-by-step guide on how to register and use Discord effectively.
```

Comment:
Use 'efficiently' instead of 'effectively' meaning that you can use the product with spending additional time or efforts.